src/model.js

```diff
@@ -422,13 +432,22 @@ const BookshelfModel = ModelBase.extend({
   morphTo(morphName) {
     if (!_.isString(morphName)) throw new Error('The `morphTo` name must be specified.');
     let columnNames, candidates;
-    if (_.isArray(arguments[1])) {
-      columnNames = arguments[1];
+    if (_.isArray(arguments[1]) || _.isNil(arguments[1])) {
+      columnNames = arguments[1] || null;
```

Comment:
As it is not a registry plugin I would check for strings instead of using `isNil`. This way the API would be intact.
```js
if (_.isArray(arguments[1]) && arguments[1].every(_.isString)) {
  columnNames = arguments[1];
```